lib/recurly/pricing/calculations.js

```diff
@@ -237,3 +237,16 @@ Calculations.prototype.planPrice = function () {
 function decimal (prop) {
   this[prop] = (Math.round(Math.max(this[prop], 0) * 100) / 100).toFixed(2);
```

Comment:
The method being used is ~36x faster, but the scientific coercion method is absolutely still within bounds of acceptable performance.

http://jsperf.com/decimal-rounding-0